playbooks/dependencies-centos-9-stream.yaml

```diff
@@ -1,3 +1,9 @@
+---
+- name: Install python3-dnf
+  become: true
+  command: dnf install -y python3-dnf
+  when: ansible_distribution_major_version | int > 38
```

Comment:
This playbook is only used for CentOS Stream 9, so I think we don't need the workaround here.